src/ContextualPopup/ContextualPopup.js

```diff
@@ -222,6 +229,18 @@ module.exports = kind(
 	initComponents: function () {
 		this.createChrome(this.tools);
 		Popup.prototype.initComponents.apply(this, arguments);
+		this.syncProperties();
+	},
+
+	/**
+	* Sync spotlightModal and modal properties on creation time.
+	*
+	* @private
+	*/
+	syncProperties: function () {
+		if (this.modal == this.spotlightModal) return;
+		if (this.hasOwnProperty('spotlightModal')) this.modal = this.spotlightModal;
```

Comment:
Do you know why we need to use `hasOwnProperty` here? I think we would still want to sync the (default) properties of the kind if they were not set on the instance.